qa-tests-backend/src/test/groovy/SACTest.groovy

```diff
@@ -158,7 +144,6 @@ class SACTest extends BaseSpecification {
         def result = DeploymentService.listDeployments()
         log.info result.toString()
         assert result.size() == 1
-        assert DeploymentService.getDeploymentWithRisk(result.first().id).hasRisk()
```

Comment:
I'm not sure if we should remove it. It was added on code review request https://github.com/stackrox/rox/pull/8591#issuecomment-871226564